src/libraries/System.Runtime.Numerics/src/System/Numerics/BigInteger.cs

```diff
@@ -1471,14 +1470,14 @@ private ReadOnlySpan<uint> ToUInt32Span(Span<uint> scratch)
 
             // Find highest significant byte and ensure high bit is 0 if positive, 1 if negative
             int msb;
-            for (msb = dwords.Length - 1; msb > 0 && dwords[msb] == highDWord; msb--);
+            for (msb = dwords.Length - 1; msb > 0 && dwords[msb] == highDWord; msb--) ;
```

Comment:
nit: Stray space was added here

Reply:
I think converting it to `while` is better, since empty statement under loop is warned by C# compiler

Reply:
> since empty statement under loop is warned by C# compiler

What warning?  We have many such loops.  If the compiler warned, the existing code wouldn't have compiled, as we enable warnings as errors.

Reply:
`while(cond) ; {}` and `for(x;x;x) ; {}` will get CS0642 for potential wrong empty statement. There's no warning here because the loop isn't follow by a `{}` block.

Reply:
Right.  So I'm not sure why that warning is relevant to this discussion.

Reply:
Well, I was meaning that "empty semicolon is a pattern that warned in some cases". I'm touching this because the old code doesn't follow IDE formatter, and I'm doing a cleanup.

Reply:
So, what's the decision on this?